kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/LogWatchCallback.java

```diff
@@ -65,23 +65,16 @@ public LogWatchCallback(Config config, OutputStream out) {
     if (out == null) {
       this.out = new PipedOutputStream();
       this.output = new PipedInputStream();
-      toClose.add(this.out);
-      toClose.add(this.output);
-    } else {
-      this.out = out;
-      this.output = null;
-    }
-
-    //We need to connect the pipe here, because onResponse might not be called in time (if log is empty)
-    //This will cause a `Pipe not connected` exception for everyone that tries to read. By always opening
-    //the pipe the user will get a ready to use inputstream, which will block until there is actually something to read.
-    if (this.out instanceof PipedOutputStream && this.output != null) {
       try {
```

Comment:
I am not sure if this is related to always enable bookmarks. Could you tell us more about this changes ?

Reply:
No, this came in unintentionally and are part of another pr.